lewis_emulators/tpg300/device.py

```diff
@@ -37,10 +70,17 @@ def _initialize_data(self):
         self.__pressure_a2 = 0.0
         self.__pressure_b1 = 0.0
         self.__pressure_b2 = 0.0
-        self.__units = None
+        self.__units = Units["mbar"]
         self.__connected = None
         self.__readstate = None
-
+        self.__switching_function_to_set = (0.0, 0, 0.0, 0, 0)
+        self.__switching_functions = [(0.0, 0, 0.0, 0, 0),
```

Comment:
I think we should add some comments describing what each of these numbers represent, or make each Tuple a class instead with named attributes so that it's obvious what the numbers represent.